roles/collect/.yamllint

```diff
@@ -1,3 +1,5 @@
+---
```

Comment:
These changes to the `.yamllint` file are due to updating molecule to a more recent version. So, these are just the newer automatic defaults from upstream.